### PR TITLE
Dispatch event on feedback response

### PR DESCRIPTION
**Pull request by prembonsall**
Sends event trigger on users selection of yes or no on feedback form.

Required for hotjar implementation on wagtail site

### CODE REVIEW

File: src/events.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/**
+ * Dispatches event named onFeedback
+ * Implement an addEventListener at the event target to utilise
+ *
+ * @param {HTMLElement} container DOM node to define event target
+ * @param {boolean} satisfied Bool to send user satisfaction
+ */
+export default function OnFeedbackEvent(container, satisfied) {
```
Comment (mikemonteith) [inline src/events.js:8]: Function names should not start with a capital letter. Only classes should be capitalised.
I'm surprised eslint didn't complain about this 🤔

Comment (prembonsall) [reply src/events.js:8]: Thought it looked off but blame this thread for throwing me https://stackoverflow.com/questions/48211891/import-functions-from-another-js-file-javascript

Comment (mikemonteith) [reply src/events.js:8]: Yeah that's an old-skool method of creating classes in javascript before the `class` keyword existed.

Technically you can do:
```
function MyClass() {
}
var instance = new MyClass();
```

**Comment by prembonsall** [comment]
Was having linting issues with polyfill and have disabled the linting on those areas. Not permanent solution was just unsure how to proceed

**Pull request merged**